examples/http-compression/http_compression.client.out

```diff
@@ -0,0 +1,63 @@
+# To invoke the autoCompress Service, use the following curl command.
+$ curl -v --output - http://localhost:9090/autoCompress -H "Accept-Encoding:deflate"
+> GET /autoCompress HTTP/1.1
+> Host: localhost:9090
+> User-Agent: curl/7.58.0
+> Accept: */*
+> Accept-Encoding:deflate
+>
+< HTTP/1.1 200 OK
+< content-type: application/json
+< content-encoding: deflate
+< content-length: 41
+<
+x��V
+* Connection #0 to host localhost left intact
+�,HU�Rr,-�WH��-(J-.���S�����	�
+
+
+# To invoke the alwaysCompress Service, use the following curl command.
+# Here the "Accept-Encoding" header is not specified.
+$ curl -v --output - http://localhost:9090/alwaysCompress/getJson
+> GET /alwaysCompress/getJson HTTP/1.1
+> Host: localhost:9090
+> User-Agent: curl/7.58.0
+> Accept: */*
+>
+< HTTP/1.1 200 OK
+< content-type: application/json
+< content-length: 49
+<
+* Connection #0 to host localhost left intact
+{"Type":"Always but constrained by content-type"}
+
+
+# To invoke the alwaysCompress Service, use the following curl command.
```

Comment:
Shall we say the Accept-Encoding header is specified in this case?

Reply:
Here the `Accept-Encoding` header is not specified. Therefore added it